ProfilesMicroService.Infrastructure/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "profiles": {
```

Comment:
I don't think that you need launchsettings in any project of type 'class library', only in Api